h2o-genmodel/src/main/java/hex/genmodel/easy/EasyPredictModelWrapper.java

```diff
@@ -526,6 +528,19 @@ public BinomialModelPrediction predictBinomial(RowData data, double offset) thro
     return p;
   }
 
+  void transformWithTargetEncoding(RowData data) {
+    Map<String, Map<String, int[]>> targetEncodingMap = getTargetEncodingMap();
+    for (Map.Entry<String, Map<String, int[]>> columnToEncodingsMap : targetEncodingMap.entrySet()) {
+      String columnName = columnToEncodingsMap.getKey();
+      String originalValue = (String) data.get(columnName); // TODO otherwise exception
+      Map<String, int[]> encodings = columnToEncodingsMap.getValue(); // Should we store encodings separately as numerator and denominator?
+      int[] correspondingNumAndDen = encodings.get(originalValue);
+      double calculatedFrequency = (double) correspondingNumAndDen[0] / correspondingNumAndDen[1];
+//      data.remove(columnName); // TODO should we remove previous entries?
```

Comment:
fix todo

Reply:
@honzasterba this was the question to anyone who can share conventions  as I don't know what are the rules/assumptions regarding immutability of the RowData that user is passing to us. cc @michalkurka